### PR TITLE
Login form

### PR DESCRIPTION
**Pull request by chrizzor**
Adding new login form and use it as standard login page

### CODE REVIEW

**Comment by tim-boerner** [comment]
When you redirect to an other start page, you need to adjust the selenium test

File: Kitodo/src/main/webapp/pages/login.xhtml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                            <h:commandLink value="#{msgs.dennochEinloggen}" id="login-go"
+                                           action="#{LoginForm.NochmalEinloggen}"/>
+                        </h:panelGroup>
+                    </h:panelGrid>
+                </sec:authorize>
+            </p:panel>
+        </p:panel>
+    </ui:define>
+
+</ui:composition>
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/pages/login.xhtml]: Missing new line at end of file.

File: Kitodo/src/main/webapp/pages/login.xhtml [hunk 2]
```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/pages/login.xhtml]: Is this line really needed? Other file did not have this line or did they miss this line?

**Comment by chrizzor** [comment]
The conflict is solved

**Pull request merged**